src/internal/tester/users.go

```diff
@@ -40,3 +41,19 @@ func LoadTestM365UserID(t *testing.T) string {
 
 	return cfg[TestCfgLoadTestUserID]
 }
+
+// expects cfg value to be a string representing an array like:
+// "['foo@example.com','bar@example.com']"
+func LoadTestM365OrgUsers(t *testing.T) []string {
+	cfg, err := readTestConfig()
+	require.NoError(t, err, "retrieving load test m365 org users from test configuration")
+
+	users := cfg[TestCfgLoadTestOrgUsers]
+	users = strings.TrimPrefix(users, "[")
```

Comment:
Feels a bit hacky, but I guess we have to go with this for now.

Reply:
Yup, agreed.